### PR TITLE
29 implement playlist read

### PR DESCRIPTION
**Pull request by haoyup**

### CODE REVIEW

File: ci/v1.1/playlist.py [hunk 1]
```diff
@@ -51,4 +51,13 @@ def get(self, p_id):
             self._url + p_id,
             headers={KEY_AUTH: self._auth}
         )
-        return r.status_code, r.json()['Playlist']  # TODO: Temporary Key
+        items = r.json()['Items']
+        return r.status_code, items
```
Comment (KamiCreed) [inline ci/v1.1/playlist.py:55]: You should do it like how it was done in `music.py` to lower redundancy if we do multiple tests. That is return the internal items individually. I'm pretty sure even though `items` is an array, there will only be one item.

File: ci/v1.1/test_playlist.py [hunk 2]
```diff
@@ -17,13 +17,26 @@ def playlist_ex(request):
...
     music_ids = [playlist_ex[0], playlist_ex[1]]
     trc, p_id = pserv.create(user_id, music_ids)
     assert trc == 200
-    trc, p_list = pserv.get(p_id)
-    assert trc == 200 #and p_list == first_playlist  # noqa E261 E262
+
+    # Test list all playlists
+    trc, items = pserv.getAll()
+    p_list = []
+    for i in items:
```
Comment (KamiCreed) [inline ci/v1.1/test_playlist.py:29]: Yea. This is unnecessary. Take a look at `music.py` to see how they return the values.

**Review by KamiCreed — CHANGES_REQUESTED** [review]
Just some minor refactoring changes.

File: ci/v1.1/playlist.py (hunk 1 above)
Comment (haoyup) [reply ci/v1.1/playlist.py]: I thought there would be multiple records with the same playlist id and different music ids, but it seems like we only have one record for one playlist id with a list of music ids. The previous implementation is to handle the case of multiple records. The code has been revised to fit our design.

**Review by KamiCreed — APPROVED** [review]

**Pull request merged**